datadog-checks-base/datadog_checks/checks/prometheus/base_check.py

```diff
@@ -101,17 +125,21 @@ def get_scraper(self, instance):
                 metrics_mapper[metric] = metric
             else:
                 metrics_mapper.update(metric)
+
         scraper.metrics_mapper = metrics_mapper
         scraper.labels_mapper = default_instance.get("labels_mapper", {})
         scraper.labels_mapper.update(instance.get("labels_mapper", {}))
         scraper.label_joins = default_instance.get("label_joins", {})
         scraper.label_joins.update(instance.get("label_joins", {}))
+        scraper.rate_metrics = self._extract_rate_metrics(default_instance.get("type_overrides", {}))
+        scraper.rate_metrics.extend(self._extract_rate_metrics(instance.get("type_overrides", {})))
         scraper.type_overrides = default_instance.get("type_overrides", {})
         scraper.type_overrides.update(instance.get("type_overrides", {}))
         scraper.exclude_labels = default_instance.get("exclude_labels", []) + instance.get("exclude_labels", [])
         scraper.extra_headers = default_instance.get("extra_headers", {})
         scraper.extra_headers.update(instance.get("extra_headers", {}))
         # For simple values instance settings overrides optional defaults
+        scraper.prometheus_metrics_prefix = instance.get("prometheus_metrics_prefix", default_instance.get("prometheus_metrics_prefix", ''))
```

Comment:
can you document the option at the bottom of https://github.com/DataDog/integrations-core/blob/master/prometheus/conf.yaml.example ?